### PR TITLE
SxG chapter11

### PR DESCRIPTION
**Pull request by sebing**

### CODE REVIEW

File: chapter11/sxg/ex_11_2.pl [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+use strict;
+
+use DateTime;
+
+if ( @ARGV == 1 ){
+	my $now = DateTime−>from_epoch( epoch => time );
+	
+    
+	my $time = DateTime->new(year => $ARGV[0], month => $ARGV[1],day > $ARGV[2]);
+
```
Comment (pratyp) [inline chapter11/sxg/ex_11_2.pl:12]: This script looks incorrect - you are getting into the conditional if there is a single argument passed to the script, but calling <code>$ARGV[0], $ARGV[1], $ARGV[2]</code> inside it. Also, wouldn't this script fail with syntax check? <code> day > $ARGV[2] </code>

File: chapter11/sxg/ex_11_2.pl [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+use DateTime;
+
+if ( @ARGV == 1 ){
```
Comment (qianliok) [inline chapter11/sxg/ex_11_2.pl:7]: Should this be `@ARGV == 3`?

Comment (sebing) [inline chapter11/sxg/ex_11_2.pl:7]: I was playing a bit.. it should be 3 :(

File: chapter11/sxg/ex_11_2.pl [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	my $now = DateTime−>from_epoch( epoch => time );
+	
+    
+	my $time = DateTime->new(year => $ARGV[0], month => $ARGV[1],day > $ARGV[2]);
```
Comment (qianliok) [inline chapter11/sxg/ex_11_2.pl:11]: `day > $ARGV[2]` I don't think this will work, I doubt if you tested this, haha :smiley:

**Pull request merged**